crates/forge/bin/cmd/bind.rs

```
@@ -267,4 +325,60 @@ impl BindArgs {
             bindings.write_to_module(bindings_root, self.single_file)
         }
     }
+
+    fn generate_alloy(&self, artifacts: &Path, bindings_root: &Path) -> Result<()> {
+        let mut solmacrogen = self.get_solmacrogen(artifacts)?;
+        println!("Generating bindings for {} contracts", solmacrogen.instances.len());
+
+        if !self.module {
+            trace!(single_file = self.single_file, "generating crate");
+            let _ = solmacrogen.write_to_crate(
+                &self.crate_name,
+                &self.crate_version,
+                bindings_root,
+                self.single_file,
+            );
+        } else {
+            trace!(single_file = self.single_file, "generating module");
+            let _ = solmacrogen.write_to_module(bindings_root, self.single_file);
```

the error is ignored